### PR TITLE
Internationalisation: Translates the panel header menu

### LINKED ISSUE #51970: Translate Panel header menu

### PR DESCRIPTION
**Pull request by eledobleefe**
**What this PR does / why we need it**:
In the process of getting ready for the translation of the portal, we need to extract strings into translation tags. One of our first steps is to translate the Dashboard in view only mode. This ticket does part of that by translating the panel header menu.

**Which issue(s) this PR fixes**:

Fixes #51970 

**Special notes for your reviewer**:

### CODE REVIEW

**Review by JoaoSilvaGrafana — COMMENTED** [review]
Just one small change. Great job on this 🎉

File: public/app/plugins/panel/graph/module.ts [hunk 1]
```diff
@@ -183,7 +184,11 @@ export class GraphCtrl extends MetricsPanelCtrl {
   }
 
   onInitPanelActions(actions: any[]) {
-    actions.push({ text: 'Toggle legend', click: 'ctrl.toggleLegend()', shortcut: 'p l' });
+    const toggleTextTranslation = t({
+      id: 'panel.header-menu.more-toggle',
```
[the remark this answers is not in the document]
Comment (eledobleefe) [reply public/app/plugins/panel/graph/module.ts:188]: Hi @JoaoSilvaGrafana! I just wanted to point out that 'Toggle legend' belongs to the "More..." menu option. But I see your point, maybe 'panel.header-menu.more-toggle-legend' or would that be too long?

[the remark this answers is not in the document]
Comment (JoaoSilvaGrafana) [reply public/app/plugins/panel/graph/module.ts:188]: Ah yes! Ok nevermind, we can leave it as is especially since Graph is deprecated 👍

**Review by JoaoSilvaGrafana — APPROVED** [review]
🎉

**Pull request merged**